src/views/splash/components/activity/activity.jsx

```diff
@@ -0,0 +1,81 @@
+var React = require('react');
```

Comment:
I don't think we want to move all these components so that they're linked to the view. That makes them seem owned by the view, so it will be confusing if we ever want to use them in a different view.  What was the reasoning to moving these all within the splash directory?